sdk/storage/azure-storage-common/src/samples/java/com/azure/storage/common/credentials/SASTokenCredentialJavaDocCodeSnippets.java

```diff
@@ -38,7 +37,7 @@ public void fromSASTokenString() {
     public void fromQueryParameters() {
         // BEGIN: com.azure.storage.common.credentials.SasTokenCredential.fromQueryParameters#Map
         SasTokenCredential credential = SasTokenCredential
-            .fromQueryParameters(Utility.parseQueryString(url.getQuery()));
+            .fromQueryParameters(StorageImplUtils.parseQueryString(url.getQuery()));
```

Comment:
Shouldn't use this in our JavaDocs as it is non-public now

Reply:
Moved them back

Reply:
parseQueryString and etc function move to StorageImplUtils
JavaDoc code snippet takes a fake query map.